thunder/benchmarks/targets.py

```diff
@@ -151,17 +152,19 @@ def interpreter_fwd(module: Callable):
     return fn_
 
 
-executors = (
-    torch_executor,
-    torch_compile_executor,
-    thunder_executor,
-)
+executors = (torch_executor, torch_compile_executor, thunder_executor)
```

Comment:
What formatter applied this change?

Reply:
@IvanYashchuk ruff configured by this https://github.com/Lightning-AI/lightning-thunder/blob/40904c762f6b1c2eb1a0c3a75a7b4b1382108b84/pyproject.toml#L84 . Let me know if in your case this doesn't match expectation.